docs/source/notes/get_start_xpu.rst

```diff
@@ -0,0 +1,417 @@
+
+Pytorch 2.4: Getting Started on Intel GPU
+=========================================
+
+The support for Intel GPUs is released alongside PyTorch v2.4.
+
+This release only supports build from source for Intel GPUs.
+
+Hardware Prerequiste
+--------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Supported Hardware
+     - Intel® Data Center GPU Max Series
+   * - Supported OS
+     - Linux
+
+
+Pytorch for Intel GPUs is compatible with Intel® Data Center GPU Max Series and Intel® Core™ Ultra Processors, and only support OS Linux with release 2.4.
+
+Software Prerequiste
+--------------------
+
+Install driver and required packages by following the `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ to Get Started.
+
+Set up Environment
+------------------
+
+Before you begin, you need to set up the environment. This can be done by sourcing the setvars.sh script provided by the ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages.
+
+.. code-block::
+
+   source ${ONEAPI_ROOT}/setvars.sh
+
+**Note**\ : the ONEAPI_ROOT is the folder you installed your ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages, usually it is /opt/intel/oneapi/ or ~/intel/oneapi/.
+
+Build from source
+-----------------
+
+Now we have all the required packages installed and environment acitvated. Follow the following command to install pytorch, torchvision, torchaudio by building from source. Refer to official guide in `pytorch from source <https://github.com/pytorch/pytorch?tab=readme-ov-file#intel-gpu-support>`_\ , `vision from source <https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#development-installation>`_ and `audio from source <https://pytorch.org/audio/main/build.linux.html>`_.
+
+.. code-block::
+
+   # Get PyTorch Source Code
+   git clone --recursive https://github.com/pytorch/pytorch
+   cd pytorch
+   git checkout main # or checkout the specific release version >= v2.4
+   git submodule sync
+   git submodule update --init --recursive
+
+   # Get required packages for compilation
+   conda install cmake ninja
+   pip install -r requirements.txt
+
+   # Pytorch for Intel GPUs only support Linux platform for now.
+   # Install the required packages for pytorch compilation
+   conda install intel::mkl-static intel::mkl-include
+
+   # (optional) If using torch.compile with inductor/triton, install the matching version of triton
+   # Run from the pytorch directory after cloning
+   # For Intel GPU support, please explicitly `export USE_XPU=1` before running command.
+   make triton
```

Comment:
You just highlighted that use must explicitly `export USE_XPU=1` and next line you don't do that. This confuses. Either it's not needed or you must actually add it:
```suggestion
   USE_XPU=1 make triton
```